src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -1,4 +1,5 @@
 import $ from 'jquery';
+// eslint-disable-next-line no-restricted-imports
 import {Flex, Box} from 'reflexbox';
```

Comment:
It would have been better to move this file to TS and remove `reflexbox` and `jquery` on the way, but I didn't want to include it in the scope of this PR